ldap/docker-compose.yml

```diff
@@ -8,14 +8,25 @@ services:
   ldap:
     image: georchestra/ldap:latest
     build: ./
+    secrets:
+      - slapd_password
+      - gs_master_passwd
     environment:
       - SLAPD_DOMAIN=georchestra.org
       - SLAPD_ORGANIZATION=georchestra
       - SLAPD_ADDITIONAL_MODULES=groupofmembers,openssh
-      - SLAPD_PASSWORD=secret
+      - SLAPD_PASSWORD_FILE=/run/secrets/slapd_password
+      - SLAPD_PASSWORD=
+      - GS_MASTER_PASSWORD_FILE=/run/secrets/gs_master_passwd
       - SLAPD_LOG_LEVEL=32768
     volumes:
       - ldap_data_test:/var/lib/ldap
       - ldap_config_test:/etc/ldap
     ports:
       - '10389:389'
+
+secrets:
```

Comment:
I was not aware that secrets were implemented in docker-compose. Will it work with classic docker / docker-compose locally ?

```
$ docker secret ls
Error response from daemon: This node is not a swarm manager. Use "docker swarm init" or "docker swarm join" to connect this node to swarm and try again.
```

Reply:
yep, it does. Docker-compose, locally, does not really store secrets, but uses the files stored in the `secrets` folder. This allows to emulate secrets behaviour.